activemodel/test/cases/validations/inclusion_validation_test.rb

```diff
@@ -158,8 +158,8 @@ def p.available_karmas
     Person.clear_validators!
   end
 
-  def test_validates_inclusion_of_with_array_value
-    Person.validates_inclusion_of :karma, in: %w( abe monkey )
+  def test_validates_inclusion_of_with_array_value_and_all_option_is_true
+    Person.validates_inclusion_of :karma, in: %w( abe monkey ), all: true
```

Comment:
Is `all` required for this test to pass? Or were you modifying an existing test on purpose 🤔

Reply:
Yes `all: true` is required, this is basically the purpose of this PR: reverting the default and making this behaviour opt-in

Reply:
@intrip What happens if we remove `all: true`?

I'm hesitant to revert the old behavior as this likely already cost some upgrade cycles for folks, to revert that behavior may be confusing. If we kept the current behavior, is there still a way to opt-out?

Reply:
@zzak 
I understand your point, honestly I've just followed https://github.com/rails/rails/issues/41051#issuecomment-757037953 suggestion. 
Making it opt-out is dead simple in any case and I don't mind following a more conservative approach.

Reply:
I will trust @rafaelfranca here, since his original suggestion was to revert to the old behavior and opt-in. 🙏 

Just making sure we didn't at least consider the conservative approach first.